### ISSUE TITLE
delta_caps

### ISSUE THREAD

**Issue by Dammi87** [sanyam5/arc-pytorch#8]
Hi Sanyam

Let me begin by thanking you for taking your time implementing this! 

I've been playing around porting your code to Tensorflow when I noticed something. The delta_caps variable in the method _get_filterbanks wants delta in the range [-1, 1] (as in zoom out, zoom in) but then in the method you take the absolute value of it.

Is there any reason for doing it like this?

**Comment by sanyam5** [comment]
There is no particular reason to take the absolute value. I took absolute value because authors do so in the paper. In fact, I believe that the network would train more easily without a `.abs` in its loss function. 

Feel free to remove it. Just be sure to modify the scaling formulas accordingly.
